generator/all_test.go

```diff
@@ -15,11 +15,44 @@
 package main_test
 
 import (
+	"bufio"
 	"errors"
+	"io/fs"
+	"os"
 	"os/exec"
+	"regexp"
+	"strings"
 	"testing"
 )
 
+var googleHeader = regexp.MustCompile(`^// Copyright 2024 Google LLC
```

Comment:
This will start failing next year, as we add files with "Copyright 2025".

Reply:
Fixed.